packages/data-model/src/models/submissions.ts

```diff
@@ -1,22 +1,42 @@
 import { relations } from 'drizzle-orm';
 import { integer, jsonb, pgEnum, pgTable, serial, timestamp, varchar } from 'drizzle-orm/pg-core';
 
-import { SchemaData, SchemaValidationError } from '@overturebio-stack/lectern-client/lib/schema-entities.js';
+import {
+	type DataRecord,
+	SchemaData,
+	SchemaValidationError,
+} from '@overturebio-stack/lectern-client/lib/schema-entities.js';
 
 import { dictionaries } from './dictionaries.js';
 import { dictionaryCategories } from './dictionary_categories.js';
 
 export const submissionStatusEnum = pgEnum('submission_status', ['OPEN', 'VALID', 'INVALID', 'CLOSED', 'COMMITTED']);
 
-export type SubmissionData = {
+export type SubmissionInsertData = {
 	batchName: string;
 	creator: string;
```

Comment:
Why do we need to track creator here but not on Update and Delete? The user should be tracked in the ActiveSubmission data.

Reply:
The `creator` field here seems to be duplicated as an Active Submission is already by `user`, as you mentioned. Will remove `creator`